examples/CMakeLists.txt

```diff
@@ -110,16 +110,15 @@ if(GINKGO_BUILD_TESTS)
     foreach(config IN LISTS config_list)
         get_filename_component(config_name "${config}" NAME_WE)
         foreach(executor IN LISTS executors)
-            add_test(NAME example_file-config-solver_${config_name}_${executor} 
-                     COMMAND 
-                     "$<TARGET_FILE:file-config-solver>"
-                     "${executor}" "${CMAKE_CURRENT_SOURCE_DIR}/file-config-solver/config/${config_name}.json"
-                     "${CMAKE_CURRENT_SOURCE_DIR}/file-config-solver/data/A.mtx"
-                     WORKING_DIRECTORY
-                     "$<TARGET_FILE_DIR:ginkgo>")
+            add_test(NAME example_file-config-solver_${config_name}_${executor}
+                COMMAND
+                "$<TARGET_FILE:file-config-solver>"
+                "${executor}" "${CMAKE_CURRENT_SOURCE_DIR}/file-config-solver/config/${config_name}.json"
+                "${CMAKE_CURRENT_SOURCE_DIR}/file-config-solver/data/A.mtx"
+                WORKING_DIRECTORY
+                "$<TARGET_FILE_DIR:ginkgo>")
             # Prevent performance issues with high core counts
```

Comment:
This doesn't actually change anything, right? I would just revert it then for a cleaner diff